src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,35 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(({
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    }) => {
+      return (
+        <div
+          className="movies"
+          key={imdbId}
+          data-cy="card"
+        >
+          <MovieCard
+            title={title}
+            description={description}
+            imgUrl={imgUrl}
+            imdbUrl={imdbUrl}
+          />
```

Comment:
I think you don't need do destructorization here
You can use movie={movie}
Because you used destructorization when you created component
Maybe it optional

Reply:
good point, thanks)

Reply:
I agree with Roman, let's pass a plain object `movie`